.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@ quickly and easily:
 
  - [ ] Make sure the PR title is formatted like:
    `[BEAM-<Jira issue #>] Description of pull request`
- - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
-       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Make sure tests pass via `mvn clean install -Pall-checks`. (Even better,
```

Comment:
I'm fine with it being an if-you-want instead of a checkbox. Top of my head wording below, feel free to ignore.

"For an extra-thorough check, instead of running `mvn clean test` try running `mvn clean install -Pall-checks`. This will run the code style checker and a few other checks. Don't worry too much -- Jenkins runs these too so you'll know if they're broken."